### PR TITLE
Fixes shower runtimes

### PR DESCRIPTION
**Pull request by silicons**

### CODE REVIEW

**Review by Cyberboss — CHANGES_REQUESTED** [review]
wash_obj should only be being called on atom/movable, find out where the bad calls are coming from and fix them

**Comment by kevinz000** [comment]
fixed, it's coming from it trying to wash the null entries in mob's heldi ndexes.

File: code/game/objects/structures/watercloset.dm [hunk 1]
```diff
@@ -317,7 +315,8 @@
 		. = 1
 		check_heat(M)
 		for(var/obj/item/I in M.held_items)
```
Comment (lzimann) [inline code/game/objects/structures/watercloset.dm]: won't a typed loop call `istype` and prevent nulls?

Comment (Cyberboss) [inline code/game/objects/structures/watercloset.dm]: Yes, this is an ineffectual change

**Comment by kevinz000** [comment]
Hmm

**Comment by kevinz000** [comment]
fixed.

**Pull request merged**